test_collections/matter/scripts/OTBR/otbr_start.sh

```
@@ -60,13 +70,36 @@ else
+RCP_DEVICE="/dev/ttyACM0"
+if [ ! -e "$RCP_DEVICE" ]; then
+	echo "ERROR: $RCP_DEVICE not found. Is the Thread RCP dongle plugged in?" >&2
+	echo "If it was just plugged in/out, wait a few seconds for it to enumerate and try again." >&2
+	exit 1
+fi
+
 print_script_step "Starting 'otbr-chip' container"
 AVAHI_PATH=$ROOT_DIR/backend/app/otbr_manager/avahi
 sudo modprobe ip6table_filter || exit 1
-sudo docker run --privileged -d --network host --name otbr-chip -e NAT64=1 -e DNS64=0 -e WEB_GUI=0 -v $AVAHI_PATH:/etc/avahi -v /dev/ttyACM0:/dev/radio $BR_IMAGE --radio-url spinel+hdlc+uart:///dev/radio?uart-baudrate=115200 -B $BR_INTERFACE || exit 1
+sudo docker run --privileged -d --network host --name otbr-chip -e NAT64=1 -e DNS64=0 -e WEB_GUI=0 -v $AVAHI_PATH:/etc/avahi -v $RCP_DEVICE:/dev/radio $BR_IMAGE --radio-url spinel+hdlc+uart:///dev/radio?uart-baudrate=115200 -B $BR_INTERFACE || exit 1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)